docs/cash-savings/Interest.md

```diff
@@ -0,0 +1,31 @@
+---
+tags: [Cash Savings]
+---
+
+# Interest
+
+## How interest is paid
+
+Interest is paid on the 1st of each month on an accrual basis. This means that interest is calculated (accrued) daily, but paid monthly. 
+
+## Interest rates
+
+Interest rates can be configured per product, split between the investor and the tenant, up to a maximum of the current [Bank of England base rate](https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate).
+
+## Rate changes
+
+Sometimes the Bank of England changes the base rate. When this happens we will use the most up to date figure and calculate the accrual for that day onwards accordingly. 
+
+## Cut offs
+
+An internal transfer must have been requested before the cut off in order to be eligible for interest. After this time interest will not be accrued until the next working day.
```

Comment:
```suggestion
Internal transfers must be requested before [insert cut off time] to be eligible for current day's interest accrual. After this time interest will not be accrued until the next working day.
```

Reply:
I've done the rewording except for the cut off time, we don't seem to document cut offs publicly in general (and we don't have one for this)